lib/tessel/wifi.js

```diff
@@ -12,9 +16,7 @@ Tessel.prototype.findAvailableNetworks = function() {
 
         var resultsJSON = '';
 
-        remoteProcess.stderr.on('data', function(d) {
-          logs.err(d.toString());
-        });
+        remoteProcess.stderr.on('data', logErr);
```

Comment:
Why isn't there a callback / promise API for remoteProcess? I'm only waiting on a single value.